config/boards/xiaomi-lmi.wip

```diff
@@ -0,0 +1,20 @@
+name: Rebuild board list in GHA menu
```

Comment:
This file doesn't look like board config file. Looks like you have placed github actions yaml in place of board config file.